openedx_events/bridge/avro_attrs_bridge.py

```diff
@@ -126,6 +122,7 @@ def _create_avro_field_definition(self, data_key, data_type):
                 " be one of the types in PYTHON_TYPE_TO_AVRO_MAPPING, an attrs decorated class, or one of the types"
                 " defined in self.extensions."
             )
+        return field
```

Comment:
Consider a single if that converts the type to a Union with “null” here instead. It may be cleaner if it isn’t missing any odd cases.